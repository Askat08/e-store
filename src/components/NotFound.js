import React from "react";
import NotFoundPic from "../404.png";

export default function NotFound() {
  return (
    <div className="container-fluid p-0">
      <img
        className="img-fluid not-found"
        src={NotFoundPic}
        alt="ERROR 404: NOT FOUND"
      />
    </div>
  );
}
