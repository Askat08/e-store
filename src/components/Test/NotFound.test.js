import React from "react";
import { shallow, mount } from "enzyme";
import NotFound from "../NotFound";

describe("NotFound component test", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<NotFound />);
  });

  it("should render Not Found component", () => {
    expect(wrapper).toMatchSnapshot();
  });

  it("should onclick invoke input callback", () => {
    const img = wrapper.find("img");
    expect(img.props().src).toEqual("404.png");
  });
});
