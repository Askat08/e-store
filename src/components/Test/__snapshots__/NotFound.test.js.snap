// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound component test should render Not Found component 1`] = `
<div
  className="container-fluid p-0"
>
  <img
    alt="ERROR 404: NOT FOUND"
    className="img-fluid not-found"
    src="404.png"
  />
</div>
`;
